files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -247,6 +251,12 @@
     - { key: "AllowTcpForwarding", value: "no" }
     - { key: "X11Forwarding", value: "no" }
     - { key: "HostKeyAlgorithms", value: "-ssh-rsa*" }
+    - { key: "MaxAuthTries", value: "6" }
+    - { key: "Ciphers", value: "chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com"}
```

Comment:
This is already the default according to `man ssh_config`. I think it's safer to let sshd use whatever the safe defaults are at the time of each sshd release. Does CIS really complain if we don't set `Ciphers`? Maybe you can double check with Chris Price and hist team if it's ok with them to ignore CIS here and let sshd choose the default list.

Reply:
@rasantel Yes, CIS does require that the settings be explicit. The rationale is that the defaults could erroneously start including insecure settings. It does put the burden on us to update this list with new settings as they become available.

Reply:
@rasantel Can I consider it resolved? feel free to reopen if required.